src/schema/v1/filter_artworks.ts

```diff
@@ -370,6 +370,8 @@ const filterArtworksTypeFactory = (
     let relayOptions: any = {}
     if (Object.keys(connectionArgs).length) {
       relayOptions = convertConnectionArgsToGravityArgs(connectionArgs)
+    } else {
+      relayOptions.size = 0
```

Comment:
Should this change also be made in v2?

Reply:
The v2 schema is already pretty different for this filter stuff, this change wouldn't necessarily make sense there.

In that schema, this resolver _is_ the connection, and so should already be requiring those explicitly. I can add a validation/error message for that case.